roles/edpm_ovn/defaults/main.yml

```diff
@@ -7,7 +7,7 @@ edpm_ovn_images_download_delay: 5
 # number of retries for download tasks
 edpm_ovn_images_download_retries: 5
 
-edpm_ovn_config_src: /var/lib/openstack/configs/ovn
+edpm_ovn_config_src: "/var/lib/openstack/configs/{{ edpm_service_name|default('ovn') }}"
```

Comment:
because you have provided a default this technically does not depend on the 
dataplane change to support edpm_service_name

minor nit i would write this as {{ edpm_service_name | default('ovn') }}